tests/test_filters.py

```diff
@@ -12,6 +12,7 @@ class TestDatetimeFilters(TestCaseBaseframe):
     def setUp(self):
         super(TestDatetimeFilters, self).setUp()
         self.now = utcnow()
+        self.today = date.today()
```

Comment:
Use a static date so that you can compare with a static string instead of the output of `strftime`.